data/ecosystem/vendors.yaml

```diff
@@ -335,3 +335,10 @@
   contact: 'https://github.com/tbragin'
   oss: true
   commercial: true
+- name: TingYun
+  distribution: true
```

Comment:
please provide evidence that there is a distribution of OpenTelemetry by TingYun (e.g. a link to a repository or within the documentation), if none is provided, set this to false

Reply:
Thanks for pointing out my mistake